packages/iov-encoding/src/decimal.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { Decimal } from "./decimal";
+
+describe("Decimal", () => {
+  describe("fromUserInput", () => {
+    it("throws helpful error message for invalid characters", () => {
+      expect(() => Decimal.fromUserInput(" 13", 5)).toThrowError(/invalid character at position 1/i);
```

Comment:
Definitely positions starting from 1 not 0?

Reply:
The idea was to allow propagating this error message into the user interface, where 1-based positions might be more intuitive. However, I don't have string opinions on this one.

Reply:
Me neither, that sounds sensible.